controllers/src/test/scala/cromwell/pipeline/controller/UserControllerTest.scala

```diff
@@ -19,18 +21,19 @@ class UserControllerTest
     with Matchers
     with ScalatestRouteTest
     with MockitoSugar
-    with PlayJsonSupport {
+    with PlayJsonSupport
```

Comment:
It's better to use composition instead of inheritance.

Reply:
Done